setup.py

```diff
@@ -27,6 +27,6 @@
       packages=['codecov'],
       include_package_data=True,
       zip_safe=True,
-      install_requires=["requests>=2.7.9", "coverage", "argparse"],
+      install_requires=["requests>=2.7.9", "coverage"],
```

Comment:
I would go with something like

```py
if sys.version_info > (3, 2) or sys.version_ifo < (2, 7):
    install_requires=["requests>=2.7.9", "coverage"],
else:
    install_requires=["requests>=2.7.9", "coverage", "argparse"],
```

This would be a much better way for this so that way it does not break for versions before 2.7.